extensions/amp-analytics/amp-analytics.md

```diff
@@ -107,6 +107,12 @@ Type attribute value: `chartbeat`
 
 Adds support for Chartbeat. More details for adding Chartbeat support can be found at [support.chartbeat.com](http://support.chartbeat.com/docs/integrations.html#amp).
 
+### ColAnalytics
+
+Type attribute value: `colanalytics`
+
+Adds support for ColAnalytics. Additionally, need the value for `id` and `colurl`.
```

Comment:
I don't see `colurl` being used anywhere. Please fix the example and the documentation.